code/src/components/Summary.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+const Summary = ({ answers, reset }) => {
+  /* const [first, intro, second, name, third, energy, fourth, genre, fifth, time] = answers; */
+  return (
+    <>
+      <div className="container-summary">
+        <div className="box-summary">
+          <h2 className="summary-text">Ok {answers[3]}!</h2>
+          <p>Let&apos;s see if we can you a some nice music based on this:</p>
+          <p>Energy: {answers[5]}</p>
+          <p>Genre: {answers[7]}</p>
+          <p>Epoke: {answers[9]}</p>
+          <p>Click on the button to play the music!</p>
+        </div>
+        <div className="result">
+          <button className="button-result" type="button">
+            <a href="https://open.spotify.com/track/6EvQkav3bbrEAWRmvs6EOj?si=53475e0f4d934c8d">
```

Comment:
Smart to add a link to Spotify!

Reply:
Yeah.. when I have time I will make it more fun and a nicer experience. :)